### PR TITLE
Add special conditions for bootable containers

### PR DESCRIPTION
**Pull request by jan-cerny**
This commit adds 2 new Jinja macros: `bootc_build` and `not_bootc_build`.  These macros define Bash conditional expressions that are evaluated as true or false if the remediation is performed during a build of a bootable container image or not performed during a build of a bootable container image. These macros can be used in Bash remediation code.  They can be used to control the remediation behavior in the bootable container build environment.

The code relies on new OpenSCAP feature implemented in https://github.com/OpenSCAP/openscap/pull/2170.

This commit changes the Bash remediation in rule
`disable_ctrlaltdel_reboot` to demonstrate usefulness of the new macros.

### CODE REVIEW

File: shared/macros/10-bash.jinja [hunk 1]
```diff
@@ -2517,3 +2517,19 @@ mkdir -p /etc/dconf/db/{{{ database }}}.d
 chmod -R u=rwX,go=rX /etc/dconf/profile
 (umask 0022 && dconf update)
 {{%- endmacro -%}}
+
+{{#
+This macro defines a conditional expression that is evaluated as true
+if the remediation is performed during a build of a bootable container image.
+#}}
+{{%- macro bootc_build() -%}}
```
Comment (Mab879) [inline shared/macros/10-bash.jinja:2525]: Should we prefix with `bash_`?

**Comment by jan-cerny** [comment]
I have add `bash_` prefix.

**Review by Mab879 — APPROVED** [review]
Thanks!

**Pull request merged**